CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this repository
```

Comment:
Let's remove

Reply:
damn apple keyboard

Let's remove "email, or any other method with the owners of this repository"

and only recommend public issues as recommended way of communication

Reply:
Sure, done.